### ISSUE TITLE
DeprecationWarning even if option is not used

### ISSUE THREAD

**Issue by xolott** [chezou/tabula-py#121]
# Summary of your issue
I'm running some tests (pytest) and every call to tabula-py. This error occurred on the method `build_options` of `wrapper.py`. I add a print line:

`print(list(kwargs.keys()))`

Which print:
`['password', 'pages', 'silent', 'format']`
# Environment

```py
Python version:
    3.6.6 (default, Jun 28 2018, 04:42:43)
[GCC 5.4.0 20160609]
Java version:
    java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)
tabula-py version: 1.3.0
platform: Linux-4.15.0-38-generic-x86_64-with-Ubuntu-18.04-bionic
uname:
    uname_result(system='Linux', node='symmetrik', release='4.15.0-38-generic', version='#41-Ubuntu SMP Wed Oct 10 10:59:38 UTC 2018', machine='x86_64', processor='x86_64')
linux_distribution: ('Ubuntu', '18.04', 'Bionic Beaver')
mac_ver: ('', ('', '', ''), '')
```

# What did you do when you faced the problem?

Normal use, happend with every PDF file

## Code:

```py
data_json = read_pdf(
                    fp,
                    output_format='json',
                    password=password,
                    pages=page,
                    silent=True,
                    java_options=["-Dsun.java2d.cmm=sun.java2d.cmm.kcms.KcmsServiceProvider"])
```

## Expected behavior:
No DeprecationWarning

## Actual behavior:

```
.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option spreadsheet.
  deprecated_option(option)
.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option nospreadsheet.
  deprecated_option(option)
```

**Comment by chezou** [comment]
Thanks for reporting the issue.

Unfortunately, I can't reproduce your issue with the following code. Could you write the whole code to reproduce it? It would be much appreciate if you can provide the PDF.

```py
In [1]: import tabula

In [2]: password = '123'

In [3]: pages = 1

In [5]: with open('./test.pdf', 'rb') as f:
   ...:     d = tabula.read_pdf(f, output_format='json', password=password, pages=pages, silent=True,java_options=["-Dsun.java2d.cmm=sun.jav
   ...: a2d.cmm.kcms.KcmsServiceProvider"])
   ...:
   ...:

In [6]: d
Out[6]:
[{'extraction_method': 'stream',
  'top': 0.0,
  'left': 0.0,
  'width': 269.0600280761719,
  'height': 104.4000015258789,
  'data': [[{'top': 77.13,
     'left': 52.86,
     'width': 25.943588256835938,
     'height': 8.789999961853027,
     'text': 'label'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''},
    {'top': 77.13,
     'left': 152.82,
     'width': 7.7891693115234375,
     'height': 8.789999961853027,
     'text': '1'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''},
    {'top': 77.13,
     'left': 207.9,
     'width': 8.057479858398438,
     'height': 8.789999961853027,
     'text': '2'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''},
    {'top': 77.13,
     'left': 262.92,
     'width': 6.139999866485596,
     'height': 8.789999961853027,
     'text': '3'}],
   [{'top': 95.61,
     'left': 52.86,
     'width': 24.254913330078125,
     'height': 8.789999961853027,
     'text': 'data'},
    {'top': 95.61,
     'left': 107.94,
     'width': 8.158851623535156,
     'height': 8.789999961853027,
     'text': 'a'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''},
    {'top': 95.61,
     'left': 162.96,
     'width': 8.366622924804688,
     'height': 8.789999961853027,
     'text': 'b'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''},
    {'top': 95.61,
     'left': 218.04,
     'width': 5.699999809265137,
     'height': 8.789999961853027,
     'text': 'c'},
    {'top': 0.0, 'left': 0.0, 'width': 0.0, 'height': 0.0, 'text': ''}]]}]
```

**Comment by xolott** [comment]
Sure,

This code behave the same with any of my 6 test PDF (from different sources of the internet)
[Here is one of them]( http://s000.tinyupload.com/?file_id=03563138359901624443)
This is the simple test:

```py
# test_tabula.py
import os
from subprocess import CalledProcessError
from tabula import read_pdf


BASE_DIR = os.path.dirname(os.path.realpath(__file__))


def get_tables(fp, password=''):
    page = 1
    tables_x_pages = []
    try:
        while 1:
            data_json = read_pdf(
                fp,
                output_format='json',
                password=password,
                pages=page,
                silent=True,
                java_options=["-Dsun.java2d.cmm=sun.java2d.cmm.kcms.KcmsServiceProvider"])
            tables_x_pages.append({
                'page': page,
                'content': data_json
            })
            page += 1
            fp.seek(0)
    except CalledProcessError:
        pass
    return tables_x_pages

def test_tabula():
    with open(os.path.join(BASE_DIR, 'pdf', 'errores', 'no_solucionado', 'InicioTramitacion5.pdf'), 'rb') as file_object:
        result = get_tables(file_object)

```

I run it with pytest:

```console
$ pytest -sv tests/test_tabula.py
================================================================================================================ test session starts ================================================================================================================
platform linux -- Python 3.6.6, pytest-3.9.1, py-1.7.0, pluggy-0.8.0 -- /media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor/.venv/bin/python3.6m
cachedir: .pytest_cache
rootdir: /media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor, inifile:
plugins: cov-2.6.0
collected 1 item

tests/test_tabula.py::test_tabula Exception in thread "main" java.lang.IndexOutOfBoundsException: Page number does not exist
        at technology.tabula.ObjectExtractor.extractPage(ObjectExtractor.java:19)
        at technology.tabula.PageIterator.next(PageIterator.java:29)
        at technology.tabula.CommandLineApp.extractFile(CommandLineApp.java:165)
        at technology.tabula.CommandLineApp.extractFileTables(CommandLineApp.java:128)
        at technology.tabula.CommandLineApp.extractTables(CommandLineApp.java:109)
        at technology.tabula.CommandLineApp.main(CommandLineApp.java:79)
Error:
PASSED

================================================================================================================= warnings summary ==================================================================================================================
/media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor/.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option spreadsheet.
  deprecated_option(option)
/media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor/.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option nospreadsheet.
  deprecated_option(option)
/media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor/.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option spreadsheet.
  deprecated_option(option)
/media/Datos/Symmetrik/DesarrolloSymmetrik/Dev/Microservice/pdf_extractor/.venv/lib/python3.6/site-packages/tabula/wrapper.py:413: DeprecationWarning: Call to deprecated option nospreadsheet.
  deprecated_option(option)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================================================================================================= 1 passed, 4 warnings in 2.83 seconds ========================================================================================================

```
PS: The `IndexOutOfBoundsException` exception is expected in this code.

## Dependencies 
Maybe this info is useful, I'm using `pipenv` as dependencies manager. This is the content of my `PipFile`:

```
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
grpcio = "==1.8.6"
pillow = "*"
six = "*"
dataclasses = "*"

[dev-packages]
grpcio-tools = "==1.8.6"
pylint = "*"
"flake8" = "*"
pydocstyle = "*"
pytest = "*"
pytest-cov = "*"
tabula-py = "*"

[requires]
python_version = "3.6"
```

**Comment by chezou** [comment]
@xolott I confirmed the issue and fixed with #122. Could you try master version?

```sh
pip install git+https://github.com/chezou/tabula-py
```
